pkg/collector/util.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package collector
+
+func Min(x, y int) int {
```

Comment:
I will remove it, but the problem is that the `math.Min` only works with float, and then we need to do a lot of conversions
`int(math.Min(float64(len(container.Command)), 10))`

Reply:
we can keep it if we have to do with lots of conversions.
I don't want to see too many stack in our code.
`int(math.Min(float64(len(container.Command)), 10))` seems 5 stacks?

`Min(len(container.Command),10)` is better.

Reply:
btw, where do we use function Min?

Reply:
There was already a logic to set the `command` size up to 10 characters....
Not sure why it was there, probably it was appearing some extreme large strings there....Don't know....

So, to decrease the command to up to 10, we can resize the slice `command[:10]`
But if the slice size is 7 for example, it will panic...
Therefore, the logic gets the `Min(len(command), 10)`

Reply:
copy?
```
_command = make([]string,10)
copy(_command, command)
```

I tested with int, if the len of source less than 10, it will fill up with 0 as make([]int,10) default value is 0.